src/features/PurchaseMembership/components/MembershipPurchaseForm.tsx

```diff
@@ -150,6 +150,17 @@ export const MembershipFormUI = ({
             amount
           );
 
+          // eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
Why is this disabled?

Reply:
According to the rule you can't have hooks inside a callback. We should migrate MembershipPurchaseForm to use the Formik hook, but out of scope for this PR imo.